### DISCUSSION TITLE
Game ideas

### DISCUSSION THREAD

**Discussion by salty-tree** [salty-tree/dgj-2024-dcspd#1] [Ideas]
Share game ideas here

**Comment by VectorScore** [comment]
game main ideas
1. 2d pixel-art platformer - exploration based
2. each 'level' is a hub based on a different environment (forest, mountain, wetland, dryland)
4. find wildlife in each hub
5. save animals through simple rpg like undertale to pass levels
6. specific gimmicks for each hub related to the environment

**Reply by edugianta** [reply]
Three biomes: 
- Grass biome
- Desert biome
- Icey tundra biome

- Grass biome: functions usual, no extra pizazz
- Desert biome: Standing on a levitating tile for too long will destroy the tile and you will slide through it. Not all tiles will have this mechanic. Sandstorm will push you back.
- Icey tundra biome: Standing still for too long will freeze. Levitating tile destruction. Low friction.

**Comment by VectorScore** [comment]
### Research topics
Main theme: SDG 15 - **Life on Land**

Target 15.3
END DESERTIFICATION AND RESTORE DEGRADED LAND
By 2030, combat desertification, restore degraded land and soil, including land affected by desertification, drought and floods, and strive to achieve a land degradation-neutral world.

Target 15.4
ENSURE CONSERVATION OF MOUNTAIN ECOSYSTEMS
By 2030, ensure the conservation of mountain ecosystems, including their biodiversity, in order to enhance their capacity to provide benefits that are essential for sustainable development.

THINGS TO DO

- Don’t shop for pets! Visit your local animal shelter and adopt an animal there.

(I have an idea here that after you complete a hub u get a pet/follower to perhaps symbolize that u adopted an animal or smth)

- Volunteer with street animal organizations or shelters: Help to sustain diversity and save lives of animal species by volunteering in organizations helping these pets as well as endangered species.

**Reply by salty-tree** [reply]
#3
